src/Types/Type.php

```diff
@@ -268,6 +268,19 @@ public static function column(BaseType|string $wrappedType): ModelColumn|EnumTyp
         return $type;
     }
 
+    public static function columnMapping(string $model): ModelColumnMapping
+    {
+        /** @var ModelColumnMapping $type */
```

Comment:
weiß jz nicht wie `static::cache` parameter ausschauen, kann man nicht diesen phpdoc nehmen als hint?
 
```
    /**
     * @template T
     *
     * @param  class-string<T> $related
     *
     * @return T
     */
```

Reply:
thanks! werd ich mir anschauen - aber ich will hier in den PR keinen commit mehr dazu reinmachen, weil ich nicht squashen werde, da schon der nächste PR wartet...